### ISSUE TITLE
NVDA fails to detect multiple linksin the same line

### ISSUE THREAD

**Issue by sukiletxe** [nvaccess/nvda#5896]
Steps to reproduce:
1. Open Word (I am using version 2007)
2. Insert two hiperlinks /either pointing to items on the same page or on the internet) in the same line, separated by one space.
3. Read the file line by line, and word by word.
4. Read character by character.

Result: In 3, NVDA behaves as if only one link was present.
Expected result: NVDA should behave as when it encounters this situation in the web. It should read every link

**Comment by ehollig** [comment]
I can confirm this in Office 2016 as well. I was able to replicate this with two links on their own line.

**Comment by Adriani90** [comment]
This is solved when using UIA in MS Word. cc: @michaelDCurran

**Comment by CyrilleB79** [comment]
@seanbudd could you add "legacy/community-support" label since it only affects Word object model?

Alternatively, you may consider that it's just a duplicate of #14956 (see ) and close one of them.
